packages/ui-date-input/src/DateInput2/props.ts

```diff
@@ -45,9 +45,9 @@ type DateInput2OwnProps = {
     nextMonthButton: string
   }
   /**
-   * Specifies the input value.
+   * Specifies the input value *before* formatting. The `formatDate` will be applied to it before displaying. Should be a valid, parsable date.
```

Comment:
I would mention here what a parseable date is, e.g. link this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#date_time_string_format

Reply:
that would be a good addition